snapshot/lib/snapshot/fixes/hardware_keyboard_fix.rb

```diff
@@ -5,8 +5,6 @@ module Fixes
 
     class HardwareKeyboardFix
       def self.patch
-        Snapshot.kill_simulator # First we need to kill the simulator
```

Comment:
It doesn't seem necessary to kill simulators so many times for each iteration of the test invocation loop. We'll do this once, apply all fixes, and then run the tests.